node/consensus/src/tests.rs

```diff
@@ -326,7 +327,8 @@ fn test_ledger_deploy() {
     let transactions = consensus.ledger.vm().speculate(sample_finalize_state(1), [transaction.clone()].iter()).unwrap();
 
     // Propose the next block.
-    let next_block = consensus.propose_next_block(&private_key, rng).unwrap();
+    let next_block =
```

Comment:
Add an assertion that the `next_block.timestamp()` matches the given median timestamp.